extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnWriterImpl.java

```diff
@@ -43,9 +43,14 @@ public class ColumnWriterImpl implements ColumnWriter {
     private static final int MIN_SLAB_SIZE = 64;
     private final SeekableByteChannel writeChannel;
     /**
-     * The buffered output stream {@link #bout} will stream data to the {@link #writeChannel}
+     * Buffered stream that writes to {@link #writeChannel}. Used for buffering small writes, particularly page headers.
      */
-    private final BufferedOutputStream bout;
+    private final BufferedOutputStream bufferedOutput;
+    /**
+     * Following is used to set the size of buffer for {@link #bufferedOutput}. In our testing, we found the page
+     * headers to be much smaller than 16 KB, so using that as the default size.
+     */
+    private static final int EXPECTED_PAGE_HEADER_SIZE = 16 << 10;
```

Comment:
I would group this with the other static constants, but not worth pushing another commit and restarting CI.

Reply:
Will add this change in my next PR